src/utils/lang/index.js

```diff
@@ -1,61 +1,3 @@
-import { isNumber } from "util";
-
-
-/**
- * Converts a value to string representation
- */
-export function convertToString(val) {
-  if (val == null) return '';
-  if (typeof val === 'string') return val;
-  if (Array.isArray(val)) return val.map(val => isNumber(val) ? val : '') + '';
-}
-
-/**
- * Converts a value into a number
- */
-export function convertToNumber(val) {
-  if (typeof val === 'number') return val;
-
-  if (isObject(val) && typeof val.valueOf === 'function') {
-    const valOf = val.valueOf();
-    val = isObject(valOf) ? valOf + '' : valOf;
-  }
-
-  if (typeof val !== 'string') {
-    return val === 0 ? val : +val;
-  }
-
-  // Remove trailing whitespaces.
-  val = val.replace(/^\s+|\s+$/g, '');
-
-  return +val;
-}
-
-/**
- * Checks if the string starts with the sub string.
- */
-export function startsWith(value, subStr) {
```

Comment:
We will be needing our custom startsWith and endWith as IE does not support these functions..

Reply:
does it apply with nodejs back-end server?

Reply:
Not needed!